tests/measurements/test_mutual_info.py

```diff
@@ -206,7 +207,7 @@ def test_qnode_mutual_info_jax_jit(self, params):
 
         params = jnp.array(params)
 
-        @qml.qnode(dev, interface="jax-jit")
+        @qml.qnode(dev, interface=interface)
```

Comment:
Why this change? It should have the same behaviour, right?

Reply:
Yes my first attempt was to add auto everywhere but the test suite time double, so I just removed the auto interface but not the fixture in case we want to readd it at some point.

Reply:
@rmoyard do you know why the test suite time doubled? Was it because of the `get_gradient_fn` being now done on call rather than on init?

Reply:
No it is because I added fixture for the interface everywhere, I would use ["auto", "jax", "jax-jit"] instead of "jax-jit" for example and therefore you triple the time for this specific test.

Reply:
ohhhh got it 😆 Thanks @rmoyard, this had me worried for a second